### ISSUE TITLE
a11y - Default sidebar - shrink/expand sidebar button - cms ui

### ISSUE THREAD

**Issue by Wagner3UB** [plone/volto#8394]
Describe the bug

The “expand/shrink” button does not receive keyboard focus. Currently, it is not possible to expand or shrink the right sidebar using only the keyboard.

Expand/shrink button - right sidebar

1. Navigate through the page using the keyboard.
2. The first keyboard-focusable element within the sidebar should be the shrink/expand button.
3. After that, the next focusable element should be the arrow button used to toggle the sidebar into full-screen mode.
4. Continue navigating and verify that focus moves to the first item outside the sidebar header, without skipping through the “Plone site” and “Blocks” elements.

Expected behavior

The shrink/expand button should:

- [ ] Be focusable using the keyboard.
- [ ] Have a visible focus indicator (border) when focused.

<img width="1919" height="931" alt="Image" src="https://github.com/user-attachments/assets/9e41b351-164d-4ae1-9ded-2449f6f497e3" />
